sklearn/tree/export.py

```diff
@@ -60,7 +60,7 @@ def _color_brew(n):
     return color_list
 
 
-def export_graphviz(decision_tree, out_file="tree.dot", max_depth=None,
+def export_graphviz(decision_tree, out_file=None, max_depth=None,
```

Comment:
oh yep fixed that now, also forgot to update the example in tree.rst, fixed that too